HelpSource/Guides/Tour-of-Special-Functions.schelp

```diff
@@ -186,26 +190,27 @@ CODE::
 (
 var plotSpecs = [
 	[   \gammaP,
-		[ [0.5,1,5,10], (0.0001, 0.05 .. 20) ],
+		[ (0.1, 0.3 .. 8), (0.01, 0.05 .. 20) ],
 		"gammaP(a, z)\nLower incomplete gamma function (normalized)\na=",
 	],
 	[   \gammaQ,
-		[ [0.5,1,5,10], (0.0001, 0.05 .. 20) ],
+		[ (0.1, 0.3 .. 8), (0.01, 0.05 .. 20) ],
 		"gammaQ(a, z)\nUpper incomplete gamma function (normalized)\na=",
 	],
 	[   \tgammaLower,
-		[ [0.5,1,5,10], (0.0001, 0.05 .. 15) ],
+		[ (0.1, 0.3 .. 8), (0.01, 0.05 .. 6) ],
 		"tgammaLower(a, z)\nLower incomplete gamma function (non-normalized)\na=",
-		0, 3,
+		0, 3
 	],
 	[   \tgammaUpper,
-		[ [0.5,1,5,10], (0.0001, 0.05 .. 15) ],
+		[ (0.1, 0.3 .. 8), (0.01, 0.05 .. 6) ],
 		"tgammaUpper(a, z)\nUpper incomplete gamma function (non-normalized)\na=",
+		0, 3
 	],
-	[	{ |delta, z| if(z.neg != delta) { tgammaDeltaRatio(z, delta) } },
-		[[-0.5, -0.2, -0.1, 0.1, 0.2, 0.5, 1], (0.0001, 0.1 .. 40)],
+	[    { |delta, z| if(z.neg != delta) { tgammaDeltaRatio(z, delta) } },
```

Comment:
You've added 4 spaces here. If you want to stick with the established style in these examples, you would need to use 3 spaces. However, changing everything to 4 spaces would be ideal.

Reply:
Fixed. for now I'll stick with the convention in the surrounding examples. More in the next conversation...